src/flow.ts

```diff
@@ -36,8 +19,8 @@ class Flow {
   constructor() {
     // Initialize handlers in the constructor
     this.buffer = [];
-    this.sessionId = null;
-    this.currentRequestId = null;
+    this.sessionId = undefined;
+    this.currentRequestId = undefined;
```

Comment:
These are already undefined, no need to set them as undefined.

Reply:
Alright. Will remove now